### PR TITLE
Add a job hunting page

### PR DESCRIPTION
**Pull request by RoseRompkxm**
Add a job-hunting page for FE, BE & Contract engineers!
Apply Email: career@venus.io

### CODE REVIEW

File: src/components/Career/JobCard.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  return (
+    <JobCardWrapper>
+      <div className="container">
+        <div className="header">
+          <img className="icon" src={icon} alt="noop" />
+          <div className="card-title">{title}</div>
+        </div>
+        <div className="content">
+          <div className="general">
+            <div className="section-title">General Infomations</div>
```
Comment (kkirka) [inline src/components/Career/JobCard.js]:
```suggestion
            <div className="section-title">General Infomation</div>
```

File: src/containers/Layout/Sidebar.js [hunk 2]
```diff
@@ -425,6 +426,14 @@ function Sidebar({ history, settings, setSetting, getGovernanceVenus }) {
             <Label primary>History</Label>
           </div>
         </NavLink>
+        <NavLink
+          className="flex flex-start align-center"
+          to="/career"
+          activeClassName="active"
+        >
+          <img className="career-icon" src={CareerIcon} alt="career" />
+          <Label primary>Career</Label>
```
Comment (kkirka) [inline src/containers/Layout/Sidebar.js:453]: I believe plural is more common, see, e.g., binance.com
Screenshot 2021-11-10 at 11 06 22

```suggestion
          <Label primary>Careers</Label>
```

File: src/components/Career/JobCard.js [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+    @media (max-width: 1200px) {
+      flex-wrap: wrap;
+    }
+  }
+`;
+
+// send mails
+function onClickApplyButton() {
+  window.location.href = 'mailto:career@venus.io';
```
Comment (kkirka) [inline src/components/Career/JobCard.js:103]: Can we add a predefined subject line for each of the positions? Sth like:

```suggestion
function onClickApplyButton(subject) {
  window.location.href = `mailto:career@venus.io?subject=${encodeURIComponent(subject)}`;
```

Not strictly a requirement (I don't think a lot of emails would come, so there won't be a mess), but nice to have to simplify our HRs' work a bit and to free the candidates of writing the subject by themselves.

File: src/containers/Layout/Sidebar.js [hunk 4]
```diff
@@ -16,6 +16,8 @@ import { getBigNumber } from 'utilities/common';
 import toast from 'components/Basic/Toast';
 import XVSIcon from 'assets/img/venus.svg';
 import XVSActiveIcon from 'assets/img/venus_active.svg';
+import CareerIcon from 'assets/img/career.png';
```
Comment (kkirka) [inline src/containers/Layout/Sidebar.js:20]: Shouldn't we have a separate icon for the non-selected "Career" menu item? Now it weirdly stands out among the others — it's a bit larger and colored. Or it is intentional? 🤔

Screenshot 2021-11-10 at 11 07 41

**Comment by JUSTISBEN** [comment]
I think also need add the address of ：
Github：https://github.com/VenusProtocol
Twitter：https://twitter.com/VenusProtocol
Medium：https://medium.com/VenusProtocol
BBS：https://community.venus.io/
Site：https://venus.io/

in Footer.

图片

How do you think?

For a temporary measure, how about to make a such change for more clear, better move the data of module "XVS" to "market":

141076260-da0df212-728f-4b03-9dc1-3c3c96bfdefeScreenshot 2021-11-19 at 08-39-22 Venus Protocol_cr

**Pull request closed without merging**